src/game/objects/index.js

```diff
@@ -3,7 +3,11 @@ export * from './DistanceUpdater';
 export * from './ObstaclesGenerator';
 export * from './Trex';
 export * from './TrexStatefull';
+// eslint-disable-next-line import/no-cycle
+export * from './King/King';
```

Comment:
```suggestion
export * from './King';
```
Jak kiedyś będziemy coś jeszcze eksportować z `King/index.js` to pewnie zapomnimy.